docs/Participation/Internal user authentication.md

```diff
@@ -4,16 +4,12 @@ In rooms where the [privacy mode](../Setup/Privacy%20modes.md) is not *open*, no
 
 ### Specification
 
-When the room receives a secret-handshake incoming connection from Alice, it checks the [internal user registry](Internal%20user%20registry.md), looking for entry in the registry corresponding to Alice's ID. If there is an entry, the room allows the incoming connection to stay alive, and grants Alice a [tunnel address](Tunnel%20addresses.md). Otherwise, the room allows the connection but does not grant Alice a tunnel address.
+When the room receives an incoming secret-handshake connection from Alice, it checks the [internal user registry](Internal%20user%20registry.md), looking for the entry corresponding to Alice's ID. If there is an entry, Alice is recognized as an internal user, granting her a [tunnel address](Tunnel%20addresses.md). Otherwise, the room recognizes Alice as an [external user](../Stakeholders/External%20user.md) and does not grant Alice a tunnel address.
 
-We need the connection to remain up even in the event of internal user authentication failing, because there are other muxrpc APIs that the room should allow external users to call, such as when consuming an invite (to become an internal user) or to perform [alias consumption](../Alias/Alias%20consumption.md).
+In either case, whether Alice is an internal or external user, the secret-handshake and muxrpc connection is allowed to remain up, because external users are allowed to [consume aliases](../Alias/Alias%20consumption.md) and create [tunneled connections](Tunneled%20connection.md) with internal users.
```

Comment:
```suggestion
In either case, whether Alice is an internal or external user, the secret-handshake and muxrpc connection is allowed to remain up, because external users are allowed to [consume aliases](../Alias/Alias%20consumption.md) and create [tunneled connections](Tunneled%20connection.md) with internal users. Unless the room is in [*Restricted* mode](../Setup/Privacy%20modes.md), in which case only internal users are allowed to connect.
```

Reply:
Thanks. I'll tweak it a tiny bit afterwards